plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/ServicesAndRoutingPage.java

```diff
@@ -226,6 +231,22 @@ protected IStatus validate() {
 				return Status.OK_STATUS;
 			}
 		});
+
+		portsViewer.getTable().addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseUp(MouseEvent event) {
+				Point pt = new Point(event.x, event.y);
+				TableItem item = portsViewer.getTable().getItem(pt);
+				if (item != null && item.getBounds(ROUTE_PORT_COLUMN_INDEX) != null && item.getBounds(ROUTE_PORT_COLUMN_INDEX).contains(pt)) {
```

Comment:
This would be even nicer if we could abstract the column-/bounds-checking away to a reusable class like TableCellMouseListener?
This could implement MouseAdapter and offer an additional #cellMouseUp(TableItem item)?